sql/telemetry_derived/experiment_enrollment_aggregates_live/view.sql

```diff
@@ -0,0 +1,91 @@
+CREATE
+OR REPLACE VIEW `moz-fx-data-shared-prod`.telemetry_derived.experiment_enrollment_aggregates_live AS (
```

Comment:
for consistency the table_id should be be quoted entirely, rather than partially:
```suggestion
OR REPLACE VIEW `moz-fx-data-shared-prod.telemetry_derived.experiment_enrollment_aggregates_live` AS (
```